services/121-service/src/registration/dto/duplicate.dto.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+// TODO: Discuss should this be name DuplicateReponseDto? (as in the guidelines) And that the api returns DuplicateReponseDto[]?
+// or should we have DuplicateDto and DuplicateResponseDto? With the latter being the response from the api which equals DuplicateDto[]
+export class DuplicateDto {
+  @ApiProperty({ example: 'Juan Garcia' })
+  public readonly name?: string;
+
+  @ApiProperty({ example: 1 })
+  public readonly registrationId: number;
+
+  @ApiProperty({ example: 1 })
+  public readonly registrationProgramId: number;
+
+  @ApiProperty({ example: 'zeeland' })
+  public readonly scope: string;
+
+  @ApiProperty({ example: ['phoneNumber'] })
+  public readonly attributeNames: string[];
+
+  @ApiProperty({ example: 'true' })
```

Comment:
I added this to help the frontend with this [frame](https://www.figma.com/design/77QfhqQ2ye51HJGD0mkBtJ/121---Portalicious?node-id=6525-527870&t=F3cbly6kXYQyQsgn-1)